src/graphql/resolvers/businessInfoResolver.ts

```diff
@@ -6,6 +6,52 @@ import {
 import { getInterpretationProvider, InterpretationConfig } from '../../providers/interpretation_providers';
 import { BusinessInferences, QueryGetBusinessInfoArgs } from '../generatedTypes';
 import { defaultConfig, DataSourceConfiguration } from '../../models/data_models/dataSourceConfiguration';
+import { BusinessInfoInput } from '../../models/data_models/types';
+import { GraphQLClient, gql } from 'graphql-request';
+
+async function fetchFromDB(
+	endpoint: string,
+	businessInfo: BusinessInfoInput
+): Promise<BusinessInferences | undefined> {
+	const query = gql`
+		query getInfo($name: String!, $address: String!) {
+			allBusinesses(where: { name: $name, address: $address }) {
```

Comment:
```suggestion
		query getInfo($businessInfo: BusinessInfoInput) {
			allBusinesses(where: $businessInfo) {
```

Reply:
The issue is that `BusinessInfoInput is not a recognized type for the db